openquake/risklib/reinsurance.py

```diff
@@ -98,7 +157,10 @@ def parse(fname, policy_idx):
             fieldmap[node['input']] = col
             continue
         treaty_type = node.get('type', 'prop')
-        assert treaty_type in ('prop', 'wxlr', 'catxl'), treaty_type
+        valid_treaty_types = ('prop', 'wxlr', 'catxl')
+        assert treaty_type in valid_treaty_types, \
```

Comment:
Instead of an assertion you can raise an InvalidFile error.

Reply:
Done